projects/questionset-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -552,7 +552,9 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
           searchFormConfig:  this.questionlibraryInput.searchFormConfig,
           metadataFormConfig: this.questionlibraryInput.metadataFormConfig
         };
-        this.questionlibraryInput.collection.eval = this.treeService.getEval();
+        if(this.treeService.getEval()) {
```

Comment:
Instead of taking this value from tree object can you set the `evalMode` in `editorConfig` take it from `editorConfig` ?
I have doubt here calling treeService will give error for Question Editor flow

Reply:
This code is only for sending evalMode property to the resource library for composite search. Here, we are not making use of this inside editor just we are passing the configuration to library this will not break anything in editor flow.
@rajnishdargan